### PR TITLE
Rxjs

### PR DESCRIPTION
**Pull request by third774**
This PR is the first part of the move to using RxJS. It only includes changes for track acquisition/retention for now, will implement the WebRTC pushing/pulling of tracks in a subsequent PR.

### CODE REVIEW

**Comment by third774** [comment]
Hah — whoops. I've deleted this app id and secret so they're no longer valid.

**Comment by threepointone** [comment]
On first glance this PR looks dope. Give me a day or so to do a full review, if it's not urgent to land?

File: app/hooks/useUserMedia.ts [hunk 1]
```diff
@@ -51,6 +50,44 @@ export default function useUserMedia(mode: Mode) {
 	const [screenshareUnavailableReason, setScreenshareUnavailableReason] =
 		useState<UserMediaError>()
 
+	const blurVideo$ = useStateObservable(blurVideo)
+	const videoEnabled$ = useStateObservable(videoEnabled)
+	const videoTrack$ = useMemo(
+		() =>
+			combineLatest([
```
Comment (threepointone) [inline app/hooks/useUserMedia.ts:57]: dope

**Review by threepointone — APPROVED** [review]
this is insane, I love it. let's just land it.

**Pull request merged**